src/features/task/components/TaskViewActions.tsx

```diff
@@ -0,0 +1,38 @@
+import changeButton from "/images/changeTaskViewButton.svg";
+import hideButton from "/images/hideTaskViewButton.svg";
+import styles from "./TaskView.module.css";
+
+interface TaskViewActionsInterface {
+  id: number;
+  title: string;
+  viewType: string;
+  onChangeViewType: (type: string, numberId: number) => void
+}
+
+export function TaskViewActions({ id, title, viewType, onChangeViewType }: TaskViewActionsInterface) {
+
+  return (
+    <>
+      {viewType === "details" ? (
+        <div className={styles.taskActions}>
+          <div className={styles.taskActionsSection}>
+            <button
+              className={styles.removeStyleDefaultButton}
+              onClick={() => console.log("Abre o modal de edição")}
+            >
+              <img src={changeButton} alt={`Change task ${title}`} />
+            </button>
+          </div>
+          <div className={styles.taskActionsSection}>
+            <button
+              className={styles.removeStyleDefaultButton}
+              onClick={() => onChangeViewType("simple", 0)}
+            >
+              <img src={hideButton} alt={`Hide task ${title}`} />
+            </button>
+          </div>
+        </div>
+      ) : null}
```

Comment:
Porque retornar nulo aqui ? Pode so usar o if viewType === "details"

Reply:
Following reason can I don't use that recommendation:
https://stackoverflow.com/questions/53048037/react-showing-0-instead-of-nothing-with-short-circuit-conditional-component